examples/ethernet_networks/update_resource.tf

```diff
@@ -10,17 +10,20 @@ provider "oneview" {
 # Updates the resource created from main.tf
 resource "oneview_ethernet_network" "ethernetnetwork" {
   name    = "TestEthNetwork_terraform_Rename"
+  ethernet_network_type = "Tagged"
   type    = "ethernet-networkV4"
   vlan_id = 100
 }
 resource "oneview_ethernet_network" "ethernetnetwork_1" {
   name    = "Auto-Ethernet-1"
+  ethernet_network_type = "Tagged"
```

Comment:
can we have example dealing with different ethernet networks like untagged, tunnel

Reply:
Done